docs/manual-CN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -149,6 +150,17 @@ nebula> SHOW CONFIGS;
 
 [[↑] 回到顶部](#常见问题)
 
+### 修改配置文件不生效
+
+**Nebula Graph** 使用如下两种方式获取配置：
+
+1. 从配置文件中（需要修改配置文件并重启服务）；
+2. 从 Meta 服务中。通过 CLI 设置，并持久化保存在 Meta 服务中，详情参考[服务器配置](../../3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md)；
+
+修改了配置文件不生效，是因为默认情况下，**Nebula Graph** 的配置参数管理采用第二种方式 (Meta)，如果希望采用第一种方式，需要在 `/home/user/nebula/build/install/etc` 配置文件 `metad.conf`、 `storaged.conf`、`graphd.conf` 中分别添加 `local_config=true` 选项。
```

Comment:
is it
``` 
local_config=true
```
or 
```
--local_config=true
```
at the top or bottom of the file?

Reply:
It's  `--local_config=true`. 3Q.